mobile/src/defaultViewComponents/defaultViewTable.js

```diff
@@ -68,24 +74,22 @@ export default class DefaultViewTable extends Component {
 
   verifyData = (isBottomTable) => {
     let items = Object.values(this.props.items)
+    items.sort(function (a,b){ 
+      return b.dateCreate - a.dateCreate
+    })
     let newItems = items.filter(item => item.isResolved === false && item.isBlock !== true)
     if (this.props.currentFilter !== "All") {
       newItems = newItems.filter(item => item.type === this.props.currentFilter.toLowerCase())
       items = items.filter(item => item.type === this.props.currentFilter.toLowerCase())
     }
-    newItems.sort(function (a,b){ 
-      return b.dateCreate - a.dateCreate
-    })
-    items.sort(function (a,b){ 
-      return b.dateCreate - a.dateCreate
-    })
     if (isBottomTable) {
       const liveItems = newItems.filter(item => item.creator.id !== client.currentUser.id)
       const resolvedItems = items.filter(item => item.isResolved)
       return liveItems.concat(resolvedItems)
     }
     else { 
-      const userItems = newItems.filter(item => item.creator.id === client.currentUser.id) 
+      const filteredItems = newItems.filter(item => item.creator.id === client.currentUser.id)
+      const userItems = filteredItems.slice()
```

Comment:
Why `slice()` the `filteredItems`?  `filter()` already creates a new array object.